projects/mci/src/mci_broadcast.c

```diff
@@ -30,6 +29,7 @@ static const uint32_t s_offset_lookup[NUM_MOTOR_CONTROLLER_BROADCAST_MEASUREMENT
   [MOTOR_CONTROLLER_BROADCAST_DSP_TEMP] = WAVESCULPTOR_MEASUREMENT_ID_DSP_BOARD_TEMPERATURE,
 };
 
+static bool should_turn_off_fan = true;
```

Comment:
It looks like this isn't used anywhere? if so, you can rid of it.
If it is needed, it should be prefixed with s_.